automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/metadata/DataSpec.scala

```diff
@@ -94,31 +95,33 @@ class DataSpec extends FreeSpec with WebBrowserSpec
-          SimpleMethodConfig.configName, SimpleMethodConfig.snapshotId, SimpleMethodConfig.configNamespace, methodConfigName)
+      withCleanBillingProject(owner) { billingProject =>
+        withWorkspace(billingProject, "TestSpec_FireCloud_launch_a_simple_workflow", aclEntries = List(AclEntry(reader.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
+          api.importMetaData(billingProject, workspaceName, "entities", TestData.SingleParticipant.participantEntity)
+          api.methodConfigurations.copyMethodConfigFromMethodRepo(billingProject, workspaceName, SimpleMethodConfig.configNamespace,
+            SimpleMethodConfig.configName, SimpleMethodConfig.snapshotId, SimpleMethodConfig.configNamespace, methodConfigName)
 
-        withSignIn(owner) { _ =>
-          val workspaceDataTab = new WorkspaceDataPage(billingProject, workspaceName).open
-          val headers1 = List("participant_id")
-          workspaceDataTab.dataTable.readColumnHeaders shouldEqual headers1
-          api.workspaces.waitForBucketReadAccess(billingProject, workspaceName)
-          val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, SimpleMethodConfig.configNamespace, methodConfigName).open
-          val submissionTab = methodConfigDetailsPage.launchAnalysis(SimpleMethodConfig.rootEntityType, TestData.SingleParticipant.entityId, "", false)
-          submissionTab.waitUntilSubmissionCompletes()
-          assert(submissionTab.verifyWorkflowSucceeded())
-          workspaceDataTab.open
-          //there is at least one filter bug - possibly two that was breaking the tests
-          //1) Not sure if bug or not: filter from launch analysis modal is still present when data tab revisited
-          //2) Filter on the datatab removes even the row being referenced
-          //this clear filter fixes the problem. Can be removed when filter bug fixed
-          workspaceDataTab.dataTable.clearFilter()
-          workspaceDataTab.dataTable.readColumnHeaders shouldEqual List("participant_id", "output")
-        }
-        withSignIn(reader) { _ =>
-          val workspaceDataTab = new WorkspaceDataPage(billingProject, workspaceName).open
-          workspaceDataTab.dataTable.readColumnHeaders shouldEqual List("participant_id", "output")
+          withSignIn(owner) { _ =>
+            val workspaceDataTab = new WorkspaceDataPage(billingProject, workspaceName).open
+            val headers1 = List("participant_id")
+            workspaceDataTab.dataTable.readColumnHeaders shouldEqual headers1
+            api.workspaces.waitForBucketReadAccess(billingProject, workspaceName)
+            val methodConfigDetailsPage = new WorkspaceMethodConfigDetailsPage(billingProject, workspaceName, SimpleMethodConfig.configNamespace, methodConfigName).open
+            val submissionTab = methodConfigDetailsPage.launchAnalysis(SimpleMethodConfig.rootEntityType, TestData.SingleParticipant.entityId, "", true)
```

Comment:
Why was this boolean changed to true?  (also in a few other locations in the file)

Reply:
it switches on call caching in an attempt to speed up the test... which doesn't actually work because each test deletes the workspace (I think)

Reply:
Yeah `withWorkspace()` does a cleanup - let's revert this.

Reply:
I left it because I was hoping one day to figure out a way to use call caching to speed this test up as we don't actually care about running the job, only handling the outputs.